### PR TITLE
Fix minor grammatical and spelling errors

### PR DESCRIPTION
**Pull request by chris-steenkamp**
- [x] I have read [freeCodeCamp's contribution guidelines](https://github.com/freeCodeCamp/freeCodeCamp/blob/master/CONTRIBUTING.md).
- [x] My pull request has a descriptive title (not a vague title like `Update index.md`)
- [x] My pull request targets the `master` branch of freeCodeCamp.
- [x] None of my changes are plagiarized from another source without proper attribution.
- [x] My article does not contain shortened URLs or affiliate links.

If your pull request closes a GitHub issue, replace the XXXXX below with the issue number.

Closes #XXXXX

### CODE REVIEW

File: guide/english/cplusplus/if-else-statement/index.md [hunk 1]
```diff
@@ -142,10 +142,10 @@ Good performance
 ### Another example of if...else if...else ladder
 Suppose we have the user input two numbers and we are going to display if either number is greater than the other. And if neither is greater than the other, then we print the statement "Both are equal".
```
Comment (martharice3000) [inline guide/english/cplusplus/if-else-statement/index.md]: Suppose we have the user input two numbers to display
```suggestion
Suppose we ask the user to input two numbers then display greater than or equal. ```

Comment (chris-steenkamp) [reply guide/english/cplusplus/if-else-statement/index.md]: The suggested sentence is much more succinct, I am just unsure as to why with the additional back-ticks which were added at the end?

**Review by thecodingaviator — APPROVED** [review]
LGTM

**Comment by thecodingaviator** [comment]
Thank you for your contribution to the page! 👍
We're happy to accept these changes, and look forward to future contributions. 📝

**Pull request merged**